static/sass/_v1_pattern_navigation.scss

```diff
@@ -0,0 +1,181 @@
+.p-navigation {
```

Comment:
To me, this seems like [abstraction for the sake of abstraction](https://www.sundoginteractive.com/blog/abstraction-for-the-sake-of-abstraction-why-add-unneeded-complexity1) when a straight import will achieve the same but I'm not precious...